src/scripts/main.js

```diff
@@ -1,3 +1,8 @@
 'use strict';
 
 // write code here
+
+const spider = document.querySelector('.spider');
+
+spider.style.insert = '0';
```

Comment:
The property 'insert' is not a valid CSS property. If you intended to set the 'inset' property, you should correct the spelling. Otherwise, ensure you are using a valid CSS property.